features.json

```diff
@@ -179,6 +179,7 @@
 				"bigInt": true,
 				"bulkMemory": true,
 				"exceptions": ["flag", "Requires flag `--experimental-wasm-eh`"],
+				"extendedConstantExprs": ["flag", "Requires flag `--experimental-wasm-extended-const`"],
```

Comment:
This is a JS flag, right? Not a Chrome flag? I think it needs to be wrapped in `--js-flags=`

Reply:
(Also we should check whether it's enabled by the `#enable-experimental-webassembly-features` flag in chrome). That would be more convenient than requiring actual command-line switches.

Reply:
This is the Node section.. so that `--js-flags` things is not needed.